docs/modules/ROOT/pages/extensions.adoc

```diff
@@ -196,3 +196,43 @@ For example, when you have defined `MyCustomFormatter` in
 ----
 $ rubocop --require ./path/to/my_custom_formatter --format MyCustomFormatter
 ----
+
+== Template language support
```

Comment:
I don't hear the term "Template language" very often. How about replacing it with the term "Template engine"?

Reply:
Since mentioning the details may seem like a nuanced difference, what about "Template support"?

The term "engine" is a nuance that refers to the processor itself that converts the template and data model into artifacts, but in reality, this feature does not support any specific processor implementation, but supports some languages that embed Ruby. For example, there is a template language called eRuby, and there are processors such as erb, erubi, and erubis that use it. In this case, I think that this feature supports the eRuby language, but not specific processor implementations.

Examples of the term "template language" include Wikipedia's [Template processor](https://en.wikipedia.org/wiki/Template_processor) page and [Slim](https://github.com/slim-template/slim)'s description. However, it is true that we are more likely to come into contact with the term "template engine" because the language and processor are often combined, and we refer exclusively to the processor when writing programs.

Reply:
Developers probably think of "Engine" when referring to ERB, Haml, Slim. OTOH, I can understand your point of view. However, this heading term is included as an anchor in the URL, so I'm curious what the intuitive terminology is for users.